.circleci/config.yml

```
@@ -7,12 +7,12 @@ orbs:
 executors:
   pre-commit-executor:
     docker:
-      - image: quay.io/astronomer/ci-pre-commit:2024-02
+      - image: quay.io/astronomer/ci-pre-commit:2025-04
         environment:
           TFENV_AUTO_INSTALL: true
   terraform-executor:
     docker:
-      - image: quay.io/astronomer/ci-terraform:2024-02
+      - image: quay.io/astronomer/ci-terraform:2025-04
```

The next month is built a few days early, and is already available.
```suggestion
      - image: quay.io/astronomer/ci-terraform:2025-05
```